walkeeper/src/safekeeper.rs

```diff
@@ -75,6 +75,10 @@ pub struct SafeKeeperState {
     /// minimal LSN which may be needed for recovery of some safekeeper (end_lsn
     /// of last record streamed to everyone)
     pub truncate_lsn: Lsn,
+    // First record starts here -- we have a hole between initial checkpoint
+    // record and the first real one, so to decode LSN of the latter should be
```

Comment:
Could you explain the meaning of `real` lsn in the comment? What's the difference between initial and real one?

Reply:
WAL after bootstrap looks like
- CHECKPOINT_SHUTDOWN at 0/01000028
- many zeros
- First record at something like 0/1699D10 

The idea is to skip zeros as decoding chokes on them. The wording could be better...